Add Country Flag Changes Updates news post

msmcculley opened the pull request:
going to pr so if we decide to merge we can have it quickly on the front page. will need a publish redate and will need some edits throughout. Woohoo! Flag changes!

## Self-check

- [x] The changes are tested against the [contribution checklist]

msmcculley committed:
add flag change newspost

msmcculley committed:
compress images for size compatibility

0x84f commented on news/2023/2023-07-25-flag-change-updates.md:
```diff
@@ -0,0 +1,41 @@
+
+![](/wiki/shared/news/2023-07-25-flag-change-updates/banner.jpg)
+
+Flag changes have been disabled from the osu! community and from osu! support since December 2021, and for the first time since then they have been reopened — with some minor adjustments.
```
This sentence reads weird, saying its "disabled" makes it sound like its a button/toggle, and not a process you go through with support

```suggestion
The flag change process for the osu! community has been frozen since December 2021, and for the first time since then it has been resumed — with some minor adjustments.
```

0x84f commented on news/2023/2023-07-25-flag-change-updates.md:
```diff
@@ -0,0 +1,41 @@
+
+Changes like these wouldn't come without some regulations, so before you get all excited and change your flag to somewhere you don't really live in, read below:
+
+- There are 6 months of previous data (meaning you can't make an account and immediately change your flag).
```
```suggestion
- There are 6 months of previous data used to determine your country of residency (meaning you can't make an account and immediately change your flag).
```

0x84f commented on news/2023/2023-07-25-flag-change-updates.md:
```diff
@@ -0,0 +1,41 @@
+Changes like these wouldn't come without some regulations, so before you get all excited and change your flag to somewhere you don't really live in, read below:
+
+- There are 6 months of previous data (meaning you can't make an account and immediately change your flag).
+- At least 4 of the 6 months have logins from only one country location, with no changes from country to country.
```
```suggestion
- At least 4 of the 6 months have to be logins from only one country location, with no changes from country to country.
```

0x84f commented on news/2023/2023-07-25-flag-change-updates.md:
```diff
@@ -0,0 +1,41 @@
+
+- There are 6 months of previous data (meaning you can't make an account and immediately change your flag).
+- At least 4 of the 6 months have logins from only one country location, with no changes from country to country.
+- You are not actively registered for any ongoing tournament.
```
Not sure what "actively" registered means, I think you can just be registered or not be registered?

```suggestion
- You must not be registered for any ongoing tournament.
```

0x84f commented on news/2023/2023-07-25-flag-change-updates.md:
```diff
@@ -0,0 +1,41 @@
+- At least 4 of the 6 months have logins from only one country location, with no changes from country to country.
+- You are not actively registered for any ongoing tournament.
+
+With some final words from ![peppy](https://osu.ppy.sh/users/2), make sure you are sure of this as once it is done, it cannot be undone.
```
I don't think this is necessary if you are just hyper linking, also reward since "make sure you are sure" doesn't read well IMO. Also not sure if the whole "final words from peppy" is necessary if you are not directly quoting him?

```suggestion
With some final words from [peppy](https://osu.ppy.sh/users/2), once you change your flag you cannot revert it back, so be 100% certain you want to do it.```

msmcculley committed:
apply the reviews

Co-authored-by: 0x84f <24534022+0x84f@users.noreply.github.com>

msmcculley committed:
remove the indirect-direct quote

msmcculley committed:
ending paragraph rewording

still felt the ending was needed so I changed it up a bit with using 0x’s review but without the extra “```” :-)

peppy commented on news/2023/2023-07-25-flag-change-updates.md:
```diff
@@ -0,0 +1,41 @@
+
+![](/wiki/shared/news/2023-07-25-flag-change-updates/user-pop-up.jpg)
+
+and subsequently, if you do not have the option to change flags it will appear like so:
+
+![](/wiki/shared/news/2023-07-25-flag-change-updates/unavailable-user-pop-up.jpg)
```
Unnecessary

peppy reviewed (commented):
i dunno. i get the push for more news content but this doesn't really feel good.

peppy commented on news/2023/2023-07-25-flag-change-updates.md:
```diff
@@ -0,0 +1,41 @@
+
+Changes like these wouldn't come without some regulations, so before you get all excited and change your flag to somewhere you don't really live in, read below:
+
+- There needs to be 6 months of previous data used to determine your country of residency (meaning you can't make an account and immediately change your flag).
+- At least 4 of the 6 months have to be logins from only one country location, with no changes from country to country.
+- You must not be registered for any ongoing tournaments.
```
Remove these.

msmcculley committed:
apply peppy reviews

removes the regulations, and changes hook and some words.

peppy commented:
> @peppy this was already discussed in-depth in the osu-news channel, we believe it would be good to cover policy changes when they make big waves in the community


We'll all I can say is that it led to a bad read so you may need more english fluent speakers as part of the discussion.

peppy commented:
This still reads bad. Likely needs someone to rewrite completely.

Walavouchey committed:
rewrite and redate

Walavouchey commented on news/2023/2023-07-28-flag-change-updates.md:
```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: "Country Flag Change Updates"
```
i think this title sounds a bit weird... how about "Enabling Country Flag Changes"?

Walavouchey commented on news/2023/2023-07-28-flag-change-updates.md:
```diff
@@ -0,0 +1,23 @@
+
+Some players may have found themselves in an awkward situation when trying to play for their country while having the wrong flag, perhaps due to moving or having created their account during a trip. In the past, this was rectifiable by contacting the [account support team](/wiki/People/Account_support_team) with proof of residency, but that system was discontinued since December 2021 due to cases ...
+
+However, this has now been changed — if you've been consistently connecting from a country that differs from the country associated with your account over a long period of time (currently this is 6 months), you will be prompted to change your country flag in the [account settings page](https://osu.ppy.sh/home/account/edit):
```
I know @peppy was adamant about not documenting the criteria but can we at least provide this much so as to not leave people guessing?

Walavouchey reviewed (commented):
rewrote basically from scratch (and took authorship due to that, but "Walavouchey & Blushing" or "osu! team" could also work here)

Blooshing commented:
> rewrote basically from scratch (and took authorship due to that, but "Walavouchey & Blushing" or "osu! team" could also work here)

I've got no qualms with the rewrite, just gives me a better idea of how stuff like this should be written going forward. I would also like to mention that I am on the side of Walavouchey & Blushing for authorship due to my commission of the banner and the skeleton/thought of the newspost, if possible.

Walavouchey committed:
change sign-off and title

Walavouchey committed:
add horizontal bar

(too little brain power rn to actually connect the paragraphs properly)

peppy committed:
Re-date and clarity reason for discontinuation
